pam/internal/adapter/nativemodel.go

```diff
@@ -404,6 +434,10 @@ func (m nativeModel) promptForChoice(title string, choices []choicePair, prompt
 	}
```

Comment:
I think these error message could be improved. IMO a simple "Invalid input" is good enough in both cases.

```diff
diff --git a/pam/internal/adapter/nativemodel.go b/pam/internal/adapter/nativemodel.go
index 6a657457..2b8b776f 100644
--- a/pam/internal/adapter/nativemodel.go
+++ b/pam/internal/adapter/nativemodel.go
@@ -361,7 +361,7 @@ func (m nativeModel) promptForNumericInputUntilValid(style pam.Style, prompt str
                return value, err
        }
 
-       err = m.sendError("Provided input can't be parsed as integer value")
+       err = m.sendError("Invalid input")
        if err != nil {
                return -1, err
        }
@@ -424,7 +424,7 @@ func (m nativeModel) promptForChoiceWithMessage(title string, message string, ch
                // TODO: Maybe add support for default selection...
 
                if idx < 1 || idx > len(choices) {
-                       if err := m.sendError("Invalid entry. Try again or input '%s'.", nativeCancelKey); err != nil {
+                       if err := m.sendError("Invalid input"); err != nil {
                                return "", err
                        }
                        continue
```

Reply:
Added this as commit 2de0fe289914423a23cdc5ff6789cba66f4d9bda